packages/kotti-ui/source/kotti-filter/components/Search.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+	<KtFieldText
+		hideValidation
+		:isLoading="isLoading"
+		:placeholder="getPlaceholder"
+		size="small"
+		:value="getSearchValue"
+		@input="handleSetSearchValue"
+	/>
+</template>
+
+<script lang="ts">
+import { KtFieldText } from '@3yourmind/kotti-ui'
+import { computed, defineComponent } from '@vue/composition-api'
+
+import { useTranslationNamespace } from '../../kotti-translation/hooks'
+import { Kotti } from '../../types'
+import { getSearchFilterInitialState } from '../utils'
+
+export default defineComponent<{
+	column: Kotti.Filter.Column
+	filter: Kotti.Filter.Filter
+	isLoading: boolean
+}>({
+	name: 'KtSearch',
+	components: {
+		KtFieldText,
+	},
+	props: {
+		column: {
+			default: () => ({}),
+			type: Object,
+		},
+		filter: {
+			default: () => ({
```

Comment:
Probably should either default to `null` (including adding `null` in the types), or be `required`